Bump Script is throwing an error and failing CI

Exception: /home/runner/work/_temp/ghwf/scripts/update-dependency.ps1:37
Line |
  37 |  …             throw "Script execution failed: $Path $action $value | ou …
     |                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | Script execution failed: scripts/update-wizard.sh get-version 
     | | output: Failed to find the plugin version in
     | /home/runner/work/sentry-react-native/sentry-react-native/scripts/../package.json
Error: Process completed with exit code 1.

https://github.com/getsentry/sentry-react-native/runs/7127617986?check_suite_focus=true
Introduced by https://github.com/getsentry/sentry-react-native/pull/2321

--- a/package.json
+++ b/package.json
@@ -49,7 +49,7 @@
     "@sentry/tracing": "7.3.1",
     "@sentry/types": "7.3.1",
     "@sentry/utils": "7.3.1",
-    "@sentry/wizard": "^1.2.17"
+    "@sentry/wizard": "1.2.17"
   },
   "devDependencies": {
     "@sentry-internal/eslint-config-sdk": "7.3.1",

